templates/inspec/tests/integration/attributes/attributes.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+project_name: 'graphite-test-sam-chef'
```

Comment:
Drive-by: should this live in source control?

Reply:
Probably not magic-modules, but some sort of source control yes. Down the line I plan on having a separate repo that will contain the VCR cassettes generated by these tests (in the future), that can also hold this sort of setup information.

Reply:
I'm alright having this live here temporarily. If it grows any longer, I think we need to figure out a better place for it to live (a storage bucket?)